pl/Regulamin_turnieju_Mistrzostwa_XXV-lecia_Heroes_of_Might_and_Magic_III.md

```diff
@@ -225,6 +225,12 @@
 
 13. Zabronione jest korzystanie przez więcej niż jedną osobę z danego konta w Mistrzostwach.
 
+14. Rozmowy między Uczestnikami w trakcie trwania Partii powinny ograniczać się jedynie do kwestii koniecznych do wypełniania postanowień Zasad Mistrzostw, chyba, że obaj Uczestnicy wyrażą zgodę na prowadzenie innej rozmowy.
+
+15. Zabrania się Uczestnikom działania na szkodę Mistrzostw i działalności Organizatorów związanej z ich przeprowadzeniem.
+
+16. Zabrania się Uczestnikom celowego przedłużania Partii i nadużywania zegara w walce.
```

Comment:
Raczej bym dał słowo "nadmiernego" zamiast "celowego" tutaj - odrobina stallu i granie pod late jest imo okej, bardziej imo chodzi o to żeby to nie było zwyczajnie zbyt mocne lub zbyt bezsensowne przeciąganie gry